docs/serverless/rules/add-exceptions.asciidoc

```diff
@@ -136,22 +136,21 @@ is only available when adding exceptions from the Alerts table.
 [[endpoint-rule-exceptions]]
 == Add {elastic-endpoint} exceptions
 
-Like detection rule exceptions, you can add Endpoint agent exceptions either by editing the Endpoint Security rule or by adding them as actions on alerts generated by the Endpoint Security rule. {elastic-endpoint} alerts have the following fields:
+Like detection rule exceptions, you can add {elastic-endpoint} exceptions either by editing the <<endpoint-protection-rules, endpoint protection rules>> or by adding exceptions as actions on alerts generated by endpoint protection rules. These alerts, known as {elastic-endpoint} alerts, have the following fields:
```

Comment:
@natasha-moore-elastic it is not possible to edit prebuilt rules at the moment. We should not advise users to do it. It will be possible in the future.